docs/reference/mapping/runtime.asciidoc

```diff
@@ -164,6 +164,32 @@ PUT my-index/
 }
 ----
 
+When no script is provided, {es} implicitly looks in `_source` at query time
+for a field with the same name as the runtime field, and returns a value if one
+exists. If a field with the same name doesn’t exist, the response doesn't
+include any values for that runtime field.
+
+In most cases, retrieve field values through
+<<doc-values,`doc_values`>> whenever possible. Accessing `doc_values` with a
+runtime field is faster than retrieving values from `_source` because of how
+data is loaded from Lucene.
+
+However, there are cases where retrieving fields from `_source` is necessary.
+For example, `text` fields do not have `doc_values` available by default, so you
+have to retrieve values from `_source`. In other instances, you might choose to
+not index specific fields or choose to disable `doc_values` on an index.
```

Comment:
the fact that you index specific fields is irrelevant as to whether you can load its values using `doc` from a script. Doc_values are required for that. Shall we shorten this sentence to "you might choose to disable doc_value on a field ?